server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -76,6 +76,11 @@ export class TenantManager {
         if (!tenant) {
             winston.error("Tenant is disabled or does not exist.");
             return Promise.reject(new Error("Tenant is disabled or does not exist."));
+        } else {
+            let externalStorageData = tenant.customData.externalStorageData;
```

Comment:
customData might be null. Maybe use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)